mindmap/mindmap.py

```diff
@@ -347,16 +370,6 @@ def get_current_mind_map(self) -> dict:
             return self.mindmap_student_body
         return self.mindmap_body
 
-    def show_staff_grading_interface(self) -> bool:
-        """
-        Return if current user is staff and not in studio.
-
-        Returns:
-            bool: True if current user is instructor and not in studio.
-        """
-        in_studio_preview = self.scope_ids.user_id is None
```

Comment:
Check yourself: `self.scope_ids.user_id` when in Studio preview returns the actual user ID